components/studio/bin/hab-studio.sh

```diff
@@ -84,8 +84,8 @@ COMMON FLAGS:
     -q  Prints less output for better use in scripts
     -v  Prints more verbose output
     -V  Prints version information
-    -D  Use a Docker Studio instead of a chroot Studio (only available on Linux)
-    -w  Use a Windows Studio instead of a Docker Studio (only available on Windows)
+    -D  Use a Docker Studio instead of a chroot Studio
+    -w  Use a Windows Studio instead of a Docker Studio (DEPRECATED)
```

Comment:
This is a Bash file... why are Windows-only options mentioned in here in the first place? 🤔

Reply:
Leftover from when there was no windows docker studio and this was the best way to tell people using the linux docker studio on a windows host how to open a windows studio. I'm just gonna remove this.